DuckDuckGo/FullscreenDaxDialogViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  FullscreenDaxDialogViewController.swift
+//  DuckDuckGo
+//
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import Core
+
+protocol FullscreenDaxDialogDelegate: NSObjectProtocol {
+
+    func hideDaxDialogs(controller: FullscreenDaxDialogViewController)
+    func closedDaxDialogs(controller: FullscreenDaxDialogViewController)
+
+}
+
+class FullscreenDaxDialogViewController: UIViewController {
+
+    @IBOutlet weak var highlightBar: UIView!
+    @IBOutlet weak var containerHeight: NSLayoutConstraint!
+    @IBOutlet weak var fullScreen: NSLayoutConstraint!
+    @IBOutlet weak var showAddressBar: NSLayoutConstraint!
+    
+    weak var daxDialogViewController: DaxDialogViewController?
+    weak var delegate: FullscreenDaxDialogDelegate?
+
+    var spec: DaxDialogs.BrowsingSpec?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        daxDialogViewController?.applyTheme(ThemeManager.shared.currentTheme)
```

Comment:
This should be encapsulated in Dax's VC `viewDidLoad()` - calling it from the outside gives no guarantee that view has been already loaded.